bitnami/harbor/templates/trivy/trivy-sts.yaml

```
@@ -39,7 +39,7 @@ spec:
         {{- end }}
     spec:
       {{- include "harbor.imagePullSecrets" . | nindent 6 }}
-      automountServiceAccountToken: {{ .Values.trivy.automountServiceAccountToken }}
+      automountServiceAccountToken: {{ .Values.trivy.serviceAccount.automountServiceAccountToken }}
```

Please keep this line as it was:
```suggestion      
      automountServiceAccountToken: {{ .Values.trivy.automountServiceAccountToken }}
```

---

Hi @migruiz4 Updated, please take a look. Thank you.